.env.local

```diff
@@ -3,9 +3,11 @@ LOCK_TIMESERIES_PERIOD_FEATURE_FLAG: true
 COMPARISON_FEATURE_FLAG: true
 AAS_LIST_FEATURE_FLAG: true
 AAS_REPO_API_URL: 'http://localhost:5064/repo'
+SUBMODEL_REPO_API_URL: 'http://localhost:5064/repo'
 MNESTIX_BACKEND_API_URL: 'http://localhost:5064'
 DISCOVERY_API_URL: 'http://localhost:5064/discovery'
 REGISTRY_API_URL: 'http://localhost:8083'
+REGISTRY_API_URL: 'http://localhost:8083'
```

Comment:
I guess this duplication was accidental?

Reply:
oops, fixed now